apps/aecore/lib/aecore/oracle/tx/oracle_registration_tx.ex

```diff
@@ -41,6 +42,7 @@ defmodule Aecore.Oracle.Tx.OracleRegistrationTx do
 
   use ExConstructor
 
+  # TODO integrate Identifiers
```

Comment:
is this todo still to be done?

Reply:
This is just a part of unused comments, will remove them soon.